custom/enikshay/duplicate_ids.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from collections import Counter
+import six
+
+from corehq.apps.users.models import CommCareUser
+from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from dimagi.utils.couch.database import iter_docs
+
+from .const import VOUCHER_ID, ENIKSHAY_ID
+
+
+def get_cases_with_duplicate_ids(domain, case_type, all_case_ids):
+    accessor = CaseAccessors(domain)
+    id_property = {'voucher': VOUCHER_ID, 'person': ENIKSHAY_ID}[case_type]
```

Comment:
Isn't below be more direct and readable?
```
id_property = VOUCER_ID if case_type == 'voucher' else ENIKSHAY_ID
```

Reply:
unless you want to raise an exception when the case type is neither `voucher` nor `person`